app/assets/stylesheets/application.css.scss.erb

```diff
@@ -43,186 +43,224 @@ td, th {
 }
 
 .near-footer {
-    border-top: 1px solid #dedede;
-    position: relative;
-    max-width: 100%;
-    padding-top: 20px;
-    padding-bottom: 20px;
-    background-image: url('<%= asset_path('nearfooter.png') %>');
+  border-top: 1px solid #dedede;
+  position: relative;
+  max-width: 100%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  background-image: url('<%= asset_path('nearfooter.png') %>');
+  text-align: center;
+  p {
+    display: block;
+    line-height: 30px;
     text-align: center;
-    p{
-        display: block;
-        line-height: 30px;
-        text-align: center;
-        width: 50%;
-        margin: 0px auto;
-        color: #666666;
-        font-size: 17px;
-    }
-    a{
-        margin: 30px 0px;
-    }
+    width: 50%;
+    margin: 0px auto;
+    color: #666666;
+    font-size: 17px;
+  }
+  a {
+    margin: 30px 0px;
+  }
 
 }
 
-
 .grey-text {
-    color: #666666;
-        }
-.event_border{
+  color: #666666;
+}
+
+.event_border {
   border: 1px solid green !important;
   margin: 3% 5% !important;
   width: 90% !important;
   padding: 20px !important;
-    border-radius: 5px;
+  border-radius: 5px;
 }
-.create_input{
-    width: 50%;
+
+.create_input {
+  width: 50%;
```

Comment:
@andela-megwali I will stop mentioning about the one-liner thingy. Check and correct appropriately.